src/EnergyPlus/OutputProcessor.cc

```diff
@@ -493,6 +493,21 @@ namespace OutputProcessor {
 
 	}
 
+
+	void 
+	SetTimeValueMinutes( Real64 minuteZone, Real64 minuteHVAC )
```

Comment:
@JasonGlazer Is this function really needed? You should be able to set TimeValue().CurMinute within the unit test. It is not part of the anonymous namespace.